README.md

```diff
@@ -4,16 +4,15 @@
 [![Coverage Status](https://coveralls.io/repos/github/observatorycontrolsystem/science-archive/badge.svg)](https://coveralls.io/github/observatorycontrolsystem/science-archive)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3ebd5b7fcff845c980f6f6a8bb4f7ab9)](https://www.codacy.com/gh/observatorycontrolsystem/science-archive?utm_source=github.com&utm_medium=referral&utm_content=observatorycontrolsystem/science-archive&utm_campaign=Badge_Grade)
 
-An application providing an API to save, retrieve, and view an observatory's science data. The data files themselves 
-are stored in AWS S3, with certain metadata for each file stored in a database for easy querying and filtering.
+An application providing an API to save, retrieve, and view an observatory's science data. The data files themselves are stored in a `FileStore` (usually S3), with certain metadata for each file stored in a database for easy querying and filtering. This application relies on the OCS [Archive library](https://github.com/observatorycontrolsystem/ocs_archive/) for configuration of its `FileStore` and input `DataFile` formats, and on the OCS [Ingester library](https://github.com/observatorycontrolsystem/ingester) to ingest file metadata into the archive and upload the files themselves to the FileStore configured.
```

Comment:
usually s3 kinda implies that it's the default...should we say Amazon s3 recommended?